cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4223,6 +4223,28 @@ pub(crate) fn define(
         .constraints(vec![WiderOrEq(Float.clone(), FloatTo.clone())]),
     );
 
+    ig.push(
```

Comment:
Reverted back to having a separate lowering for fvdemote after some offline discussion with @abrown. The impact of adding a constraint was not clear. For example, I thought if adding the constraint:

.constraints(vec![WiderOrEq(Float.clone(), FloatTo.clone())]),

this meant that x needed to be wider or equal to a. When I did this the spec test passed as expected. However, when I reversed the constraint to say

.constraints(vec![WiderOrEq(FloatTo.clone(), Float.clone())]),

This also passed. For this reason I just left it out which is what the original patch submission did (as written in the notes).